### ISSUE TITLE
bob_plugin.py doesn't work in blender 2.8

### ISSUE THREAD

**Issue by Leonkoithara** [Mrmaxmeier/BombSquad-Community-Mod-Manager#76]
It seems that they have made a lot of changes to the blender api in 2.8 e.g register_module(__name__) no longer works

**Comment by shantanuaryan67** [comment]
Hey. I've tried to update the bob plugin. It is a bit buggy and mostly works well.
If anyone has the original source code, I can look into it and make it better.
I am also working on another addon that will improve the workflow of creating models for the game.

forked repo: https://github.com/shantanuaryan67/BombSquad-Community-Mod-Manager/tree/master/utils/blender_2-80

**Comment by Rishav-G** [comment]
@shantanuaryan67 i am having an error while exporting bob files.
![2020-04-18](https://user-images.githubusercontent.com/43693024/79641487-7332e380-81b5-11ea-96a6-5bc0ad65ebff.png)

**Comment by shantanuaryan67** [comment]
@Rishav-G . Oh, that's funny. The error only shows up when exporting bob file with the old method. When using batch import, I get no errors. Also, the same line of code is used in exporting cob, but no errors in that

**Comment by Rishav-G** [comment]
@shantanuaryan67 so any solution for the problem??
And please can you explain how to do batch import ? I am unable to understand that

**Comment by shantanuaryan67** [comment]
@Rishav-G bob export should work now however cob export is still broken. to use batch import, make sure you have both addons installed, then in 3d viewport you will get a bombsquad panel in create tab. just select the models directory and enter 'neospaz' as the name. click import and you should get the entire character assembled

**Comment by Rishav-G** [comment]
@shantanuaryan67 so sorry i misunderstood batch import with export.

Thanks for such speedy work.
So now exporting bob is working but not cob??

**Comment by shantanuaryan67** [comment]
@Rishav-G yes

**Comment by Rishav-G** [comment]
@shantanuaryan67 i know u told me exporting cob has some error but my model cob was exported successfully but in game it gave me this error.
![error22](https://user-images.githubusercontent.com/43693024/79683845-76c47a00-824a-11ea-8a3b-4f9ee260b057.png)

**Comment by shantanuaryan67** [comment]
@Rishav-G yes. it only exports the vertices not faces. therefore no error in blender but the game throws error

**Comment by Rishav-G** [comment]
@shantanuaryan67 ok.
So are you working on this error??
Eagerly waiting for this to get solved as i dont know anything about coding.

**Comment by Rishav-G** [comment]
@shantanuaryan67 any updates?

**Comment by SmooothSoundz** [comment]
I just use Blender 2.79b...

**Comment by TheMikirog** [comment]
> I just use Blender 2.79b...

_"Oh, screw the guys who like the new interface and navigation of the new Blender in comparison to the old one and don't want to jump between two different versions just to make a BombSquad map!"_

**Comment by SmooothSoundz** [comment]
> > I just use Blender 2.79b...
> 
> _"Oh, screw the guys who like the new interface and navigation of the new Blender in comparison to the old one and don't want to jump between two different versions just to make a BombSquad map!"_

![image](https://user-images.githubusercontent.com/47755225/80922727-ef4c3000-8d4c-11ea-8599-b6253658221f.png)

Maybe you could help fix the cob issue...

**Comment by shantanuaryan67** [comment]
@Rishav-G @TheMikirog I have updated the script. Everything should work now.
